raiden_contracts/data/source/services/UserDeposit.sol

```diff
@@ -161,7 +161,7 @@ contract UserDeposit is Utils {
 
         withdraw_plans[msg.sender] = WithdrawPlan({
             amount: amount,
-            withdraw_timestamp: block.timestamp + withdraw_delay
+            withdrawable_after: block.timestamp + withdraw_delay
```

Comment:
Per the description of the issue, shouldn't we also rename `withdraw_delay` to `withdraw_timeout`? Does this variable match the `_timeout` usecase, or is it something different?

Reply:
Good call ✅